src/ain-db/ref.ts

```diff
@@ -246,7 +247,7 @@ export default class Reference {
 
   /**
    * Returns the owner evaluation result.
-   * @param params 
+   * @param params
```

Comment:
One thing that needs to be updated is 'permission' parameter to evalOwner(), which is the kind of permission to evaluate with like "write_rule" and "branch_owner".
e.g.)
{
  "address": "0x00...",
  "ref": "/apps/an_app",
  "permission": "branch_owner"
}

Reply:
Thanks for pointing that out! Updated to include a permission param.